src/Compilers/Test/Core/CommonTestBase.cs

```diff
@@ -46,6 +46,7 @@ public abstract partial class CommonTestBase : TestBase
             Action<IModuleSymbol> symbolValidator = null,
             SignatureDescription[] expectedSignatures = null,
             string expectedOutput = null,
+            bool trimOutput = true,
```

Comment:
needed to add this as many interpolation tests are very explicitly validating that the right whitespace is emitted.